api.web.yaml

```diff
@@ -285,6 +285,24 @@ paths:
         default:
           description: Unexpected error
 
+  /studies/pending:
```

Comment:
I think `pending` should _probably_ be a query parameter on `/studies` as it's a filter